OptimizelySDK.NetStandard16/OptimizelySDK.NetStandard16.csproj

```diff
@@ -53,6 +54,7 @@
         <Compile Include="..\OptimizelySDK\ProjectConfig.cs" />
         <Compile Include="..\OptimizelySDK\Utils\EventTagUtils.cs" />
         <Compile Include="..\OptimizelySDK\Utils\Validator.cs" />
+        <Compile Include="..\OptimizelySDK.NetStandard16\Properties\AssemblyInfo.cs" />
```

Comment:
Can you try if it works. 
`.\Properties\AssemblyInfo.cs`
instead of 
`..\OptimizelySDK.Net`